Firmware/RTK_Surveyor/menuMessages.ino

```diff
@@ -223,18 +223,18 @@ void menuMessages()
   bool response = setMessages(); //Does a complete open/closed val set
   if (response == false)
   {
-    systemPrintln("menuMessages: Failed to enable UART1 messages - Try 1");
+    systemPrintln("menuMessages: Failed to enable messages - Try 1");
 
     response = setMessages(); //Does a complete open/closed val set
 
     if (response == false)
-      systemPrintln("menuMessages: Failed to enable UART1 messages - Try 2");
+      systemPrintln("menuMessages: Failed to enable messages - Try 2");
     else
-      systemPrintln("menuMessages: UART1 messages successfully enabled");
+      systemPrintln("menuMessages: messages successfully enabled");
```

Comment:
Messages should be capitalized to match other error/status prints.

Reply:
I deleted UART1 because on the Ref Stn they are actually output on SPI, not UART1.

Reply:
Yep. I believe it should be "menuMessages: **M**essages successfully enabled" to match other statements.

Reply:
Ah. Indeed. Oh, go on then...